README.md

```diff
@@ -381,6 +381,55 @@ const hubspot = new Hubspot(params)
 return hubspot.oauth.getAccessToken(params).then(...)
 ```
 
+## Contributing
+
+Before submitting a pull request, please make sure the following is done:
+
+1. Fork the repository and create your branch from master.
+1. Run npm install in the repository root.
+1. If you’ve fixed a bug or added code that should be tested, add tests!
+1. Ensure the test suite passes (npm test). Tip: `npm run mocha -- --grep "test name"` is helpful in development.
+1. Format your code with prettier and eslint (npm run lint).
+
+## Development Workflow
+
+After cloning this repo, run `npm install` to fetch dependencies. Then run the
+test suite with `npm test`. From master, this should be green. Our tests mock
+out requests to HubSpot's api using [nock]. We then recommend running the test
+suite without mocking api requests with `NOCK_OFF=true npm run mocha`. This
+should fail as you'll need some [environment variables] for real requests to
+pass.
+
+[nock]: https://github.com/nock/nock
+[environment variables]: https://github.com/motdotla/dotenv
+
+If you haven't already, create a [developer account] on hubspot. You'll want to
+[create an app] as well. Then, create a new file, `.env` in the root of the
+repo. Inside you'll need to add an [app id], a HubSpot user id, an [api key] and
+an [oauth access token];
+
+NOTE: Your HubSpot user ID; This can be found in the same place as your
+Developer HAPIkey in your Developer portal.
+
+[developer account]: https://developer.hubspot.com
+[create a app]: https://developers.hubspot.com/docs/faq/how-do-i-create-an-app-in-hubspot
+[app id]: https://developers.hubspot.com/docs/faq/how-do-i-find-the-app-id
+[oauth access token]: https://developers.hubspot.com/docs/methods/oauth2/oauth2-overview
+
+```.env
+API_KEY="some-key"
```

Comment:
Can we make these values more representative of the shape of real data? I'd prefer they not be valid, but have the same format.

Reply:
Done in 
https://github.com/MadKudu/node-hubspot/pull/138/commits/79d96dc9dfd68f98b68c89a30d2442dd0fe55244